config/management/src/lib.rs

```diff
@@ -207,7 +207,7 @@ pub struct SecureBackends {
     /// The local secure backend, this is the source of data. Secure
     /// backends are represented as a semi-colon deliminted key value
     /// pair: "k0=v0;k1=v1;...".  The current supported formats are:
-    ///     Vault: "backend=vault;server=URL;token=TOKEN"
+    ///     Vault: "backend=vault;server=URL;token=PATH_TO_TOKEN"
```

Comment:
nit : are we not supporting both the PATH_TO_TOKEN and the TOKEN?

Reply:
Not for this tool, I was told explicitly if you let them use it this way, they will mess up, they will accidentally share their secrets :P

Reply:
e.g., here's the command I used -- oh there's my token ooops